docroot/themes/custom/yukonca_glider/src/sass/drupal/paragraphs/_secondary-content.paragraph.scss

```diff
@@ -0,0 +1,14 @@
+.secondary-content-item {
+  a {
```

Comment:
Why is this part not a pattern?

Reply:
Actually these styles are already applies in the column_layout_card pattern.  This scss file should have been removed.